go.mod

```diff
@@ -23,20 +23,28 @@ require (
 )
 
 require (
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
```

Comment:
go.mod and go.sum files aren't updated. Please run `go mod tidy` again.

Reply:
sure